lib/screens/scan_and_convert.dart

```diff
@@ -53,7 +60,11 @@ class _ScanAndConvertState extends State<ScanAndConvert> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: const Text('Image to convert'),
+        backgroundColor: primaryColor,
```

Comment:
remove ```backgroundColor``` because theming is maintained from main

Reply:
Done